Counter.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Please export a `counter` function so we can create multiple instances of a counter.

If you don't want to use `.prototype`, feel free to define a function that returns a new instance using `Object.create(prototype)`.